.travis.yml

```diff
@@ -26,24 +26,23 @@ cache:
     - node_modules
     - $HOME/.gimme
 
-services:
-  - mysql
-  - memcached
-  - elasticsearch
-  - redis
-
 addons:
   apt:
     sources:
       - elasticsearch-5.x
     packages:
       - swig
-      - oracle-java8-set-default
       - elasticsearch
       - gettext
       - librsvg2-bin
       - pngcrush
 
+services:
+  - mysql
+  - memcached
+  - elasticsearch
+  - redis
+
```

Comment:
Is this movement in the file deliberate?

Reply:
More homeopathic, I like this structure more but I also read that the structure of the travis file does have some impact on how things are running. I doubt that, but now it's in my head :laughing: